### PR TITLE
More config parameters for MoveToMap

### PR DESCRIPTION
**Pull request by DBa2016**
## Short Description:
Moved module-level constants (snipe parameters) into the config file, see config.json.map.example

### CODE REVIEW

**Comment by mjmadsen** [comment]
@DBa2016 thanks! Could you update configuration_files.md as well?

**Comment by DBa2016** [comment]
I updated the example file, will look into the other one later today (in the evening I think)

**Comment by mjmadsen** [comment]
Sounds good. Do you want to wait to merge until that's done? I know you dislike how our documentation is severely outdated.

File: pokemongo_bot/cell_workers/move_to_map_pokemon.py [hunk 1]
```diff
@@ -70,27 +70,10 @@
...
-UPDATE_MAP_MIN_TIME_SEC = 120
-
-# Number of seconds to sleep between teleporting to a snipped Pokemon.
-SNIPE_SLEEP_SEC = 2
-
-# Max Sniping in chain
-SNIPE_MAX_IN_CHAIN = 2
-
-# Don't call sniper every time in workers
-SNIPE_SKIP_IN_ROUND = 5
```
Comment (solderzzc) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: Default to 30 will be safer

**Comment by mjmadsen** [comment]
@DBa2016 @solderzzc Since he plans to work on it later (updating docs), and the default value is still the same, I'm going to merge.

**Comment by mjmadsen** [comment]
:+1:

**Comment by solderzzc** [comment]
Sure, Let me modify the default value

**Comment by mjmadsen** [comment]
Thanks @solderzzc

**Pull request merged**